docker/build.sh

```diff
@@ -0,0 +1,9 @@
+#! /bin/bash
```

Comment:
There's already a `docker-build.sh` file in the root of the repo. Should just add the build to there, and as I said in another comment, remove the version. It's a test image, let's have it track latest for now and worry about versioning once we do backwards incompatible changes.

Reply:
It seems like there *have* been backward-incompatible changes?  I'm concerned that matching latest means that clients will be built to expect unreleased features. This might already be the case with the rust client?

Reply:
(Well, perhaps those are *forward-incompatible* changes?)

Reply:
Also, I don't see `docker-build.sh`, but I can move this file to that name.

Reply:
Forward incompatible. Everything should be backwards compatible. As for the file: my bad. It's my own style but it's not actually there in the public repos. Keep the file here, that's fine.